Makefile

```diff
@@ -221,3 +222,44 @@ help:
 	@printf "\n%s\n\n" "usage: make <target>"
 	@grep -E '^[0-9a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[0;36m%-30s\033[0m %s\n", $$1, $$2}'
 
+
+# Utility targets
+#######################
+
+jq-installed:
+ifeq ($(OS),Darwin)
+	# Skipping installation of jq for local dev on Mac.
+	# You can install via 'brew install jq' the following command if needed.
+else
+	which jq ; if [ $$? -eq 1 ] ; then sudo apt-get install -y jq ; fi
+endif
+
+# BSD and GNU cross-platfrom sed -i ''
+SEDVERSION = $(shell sed --version >/dev/null 2>&1 ; echo $$? )
+
+ifeq ($(SEDVERSION), 1)
+# BSD sed requires a space and does not support --version
+SEDI = sed -E -i ''
+else
+# GNU sed forbids a space and supports --version
+SEDI = sed -E -i''
+endif
+
+
+# Code change targets
+#######################
+
+SYFT_LATEST_VERSION = $(shell curl "https://api.github.com/repos/anchore/syft/releases/latest" 2>/dev/null | jq -r '.tag_name')
+# Regex note: double-dollarsign for Makefile escaping; % instead of / for Sed to match anchore/syft
+latest-syft: jq-installed
```

Comment:
Would it be better to describe the task with a verb and noun, to match more closely what's going on?

update-grype
update-syft
update-anchore-tools

Reply:
> I tested and it works. It would be nice to figure out how to add the targets to the printout when you run bare make for discovery purposes, but maybe that's a good follow up when we have spare time.

Good catch @rbrady! To add this to the help output, you just add two `##` after the target declaration. Like this:
```
latest-syf: jq-installed ## Update syft to the latest version
```

Reply:
Nice. I'll make these improvements.